kodisync.js

```diff
@@ -54,11 +54,25 @@ async function wait(ms) {
 function playingSameThing(hosts) {
 	if (hosts.length < 2) return true;
 	if (hosts.some((host) => host.playerid == null)) return false;
-	return !hosts.some((host, index) => {
-		if (index === 0) return false;
-		return host.currentItem.showtitle !== hosts[0].currentItem.showtitle
-			|| host.currentItem.season !== hosts[0].currentItem.season
-			|| host.currentItem.episode !== hosts[0].currentItem.episode;
+	return hosts.every((host, index) => {
+		if (index === 0) return true;
+
+		// Check if showtitle, season, and episode information is available (non-empty and not -1)
+		const showInfoAvailable = (host.currentItem.showtitle && hosts[0].currentItem.showtitle)
```

Comment:
```suggestion
		const showInfoAvailable = host.currentItem.showtitle && hosts[0].currentItem.showtitle
```

These parens don't do anything, do they?